content/docs/cml/self-hosted-runners.md

```diff
@@ -79,31 +79,31 @@ We like the
+| `--repo <URL>`                    | Inferred from environment variables                        | Git repository URL e.g. https://github.com/iterative/cml                                                                                              |
+| `--token <token>`                 | Inferred from environment variables                        | Personal access token with enough permissions to register a self-hosted runner on the repository                                                      |
+| `--cloud <vendor>`                | N/A                                                        | Cloud provider to deploy the runner. Accepts `aws` or `azure`. If not set, the runner will be deployed locally and `--cloud-` options will be ignored |
+| `--cloud-region <region>`         | `us-west`                                                  | Region where the instance is deployed. Accepts `us-east`, `us-west`, `eu-west`, `eu-north` or native regions                                          |
+| `--cloud-type <type>`             | N/A                                                        | Instance type. Accepts `m`, `l`, `xl` or native types like `t2.micro`                                                                                 |
+| `--cloud-gpu <type>`              | `nogpu`                                                    | GPU type. Accepts `nogpu`, `k80` or `tesla`.                                                                                                          |
+| `--cloud-hdd-size <size>`         | N/A                                                        | Storage size in GB                                                                                                                                    |
+| `--cloud-ssh-private <key>`       | Automatically generated throwaway key                      | Custom private RSA SSH key                                                                                                                            |
+| `--cloud-spot`                    | N/A                                                        | Request a spot instance. If not set, `--cloud-spot-price` won't have any effect                                                                       |
+| `--cloud-spot-price <price>`      | Current spot bidding price                                 | Maximum spot instance bidding price in USD                                                                                                            |
+| `--cloud-startup-script <script>` | N/A                                                        | Run the provided Base64-encoded Linux shell script during the instance initialization                                                                 |
```

Comment:
```suggestion
| `--cloud-startup-script <text>`   | N/A                                                        | Run the provided Base64-encoded Linux shell script during the instance initialization                                                                 |
```

Reply:
or `<base64>`. Think of users who will only see this part in the `--help` output and need a hint as to what kind of values the option arg accepts.

Reply:
```suggestion
| `--cloud-startup-script <base64>` | N/A                                                        | Run the provided Base64-encoded Linux shell script during the instance initialization                                                                 |
```

I'd go for `base64` as it seems go be higher in the usefulness/byte scale. 😄